user-guide/docs/tools/visualization/hazmapper.md

```diff
@@ -1,514 +1,525 @@
 ## Hazmapper
```

Comment:
Minor concern. This page exists on it's own, first heading should be an `h1` (`# …` not `## …`) and so on. If no time to do this, just tell me so. No prob.